src/arch/x86/Makefile.inc

```diff
@@ -275,6 +275,7 @@ postcar-$(CONFIG_EARLY_EBDA_INIT) += ebda.c
 postcar-$(CONFIG_IDT_IN_EVERY_STAGE) += exception.c
 postcar-$(CONFIG_IDT_IN_EVERY_STAGE) += idt.S
 postcar-y += exit_car.S
+postcar-y += ../../mainboard/$(MAINBOARDDIR)/bios_knobs.c
```

Comment:
@piotr-kleins why here? add this line in mainboard/pcengines/apu1/Makefile.inc

Reply:
Fixed